### ISSUE TITLE
YouTube changed protocol? 

### ISSUE THREAD

**Issue by degarb** [bravepipeproject/BravePipeExtractor#22]
Both bravenewpipe and newpipe no longer can download YouTube as of today.

The typical malformed url complaint whenever youtu.be changes something.

**Comment by evermind-zz** [comment]
Could you be more descriptive. Which video for example?

**Comment by strider72** [comment]
As of yesterday or so, neither NewPipe or BraveNewPipe can do YouTube. Keep a close eye on NP for a fix I guess...?

**Comment by degarb** [comment]
This topic got fixed.  However, there has been a new update to youtu.be at the tail end of January 2022, and NewPipe and bravenewpipe quite working on all of my phones.

NewPipe fixed this on February 2, with .20.16

I hope bravenewpipe will too.  I prefer bravenewpipe because they support more platforms. I also found that by having both on the phone, it is more, 2x, likely that I will find a link towards the top to rip the audio or video.  Android randomly sorts the links on the share menu which is annoying when you have 200 applications installed on a modern phone.

I do not see any odyssee porn, nor do I see any support for it.  Libry has porn, but Odysee is a totally different service than libry.   Odysee has the best alternative feed than youtu.be, so I can watch my doctors and scientists without youtu.be censoring science in the name of fauci or global warming or anti fission, only solar or wind agendas.
